python/ray/rllib/rollout.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
```

Comment:
I believe this is very similar to `eval.py` - perhaps just modify that instead of introducing a new file?

Reply:
that's reasonable; `eval.py` is missing `ModelCatalog.get_preprocessor_as_wrapper` though, right?

Reply:
yeah, the wrapper would be a nice addition